official/vision/modeling/backbones/resnet.py

```diff
@@ -129,6 +129,7 @@ def __init__(
       kernel_regularizer: Optional[tf.keras.regularizers.Regularizer] = None,
       bias_regularizer: Optional[tf.keras.regularizers.Regularizer] = None,
       bn_trainable: bool = True,
+      use_first_projection: bool = True,
```

Comment:
Can you also update the input arguments to build_resnet function in line 418?

Reply:
sure, done